src/donate.hbs

```diff
@@ -0,0 +1,72 @@
+---
+title: Donate
+description: Please support our continued work to develop products for student success
+prices: [1, 5, 10, 25, 50, 100]
+---
+
+<section class="general section is-revealing">
+	<div class="general-inner section-inner">
+		<div class="container-sm">
+			<h1>Buy us a Coffee</h1>
+			<p>Josh's pithy pitch goes here to make you give us tons of money.</p>
+			<div class="text-center m-auto">
+				<div class="mt-64 switch-field">
+					{{#each prices as |price|}}
+						<input type="radio" name="prices" id="price-{{price}}" value="{{price}}" />
+						<label for="price-{{price}}">{{price}}</label>
+					{{/each}}
+					<input type="radio" name="prices" id="price-other" value="other" />
+					<label for="price-other">Other</label>
+				</div>
+				<div id="slider-value" class="m-auto pb-32">
+					<h2 class="mt-16" id="coffee-count">$10</h2>
+					<input id="coffee-slider" class="m-auto" type="range" value="10" min="1" max="100" />
+				</div>
+				<a class="m-auto button button-primary button-wide-mobile" id="donate-button">Buy us a Coffee</a>
+				<div class="mt-16 mb-32 payment-options">
+					<img src="/static/images/pm-visa.svg" />
+					<img src="/static/images/pm-mastercard.svg" />
+					<img src="/static/images/pm-amex.svg" />
+					<img src="/static/images/pm-discover.svg" />
+					<img src="/static/images/pm-googlepay.svg" />
+					<img src="/static/images/pm-applepay.svg" />
+				</div>
+				<small>or venmo us at <a id="donate-venmo" href="https://venmo.com/?txn=pay&audience=friends&recipients=gradebook&note=Buy%20the%20dev%20team%20some%20coffee">@gradebook</a></small>
+			</div>
+		</div>
+	</div>
+</section>
+
+{{#block "footer"}}
+<script src="https://js.stripe.com/v3/"></script>
+<script>
+	stripe = Stripe('{{stripe.publishable}}');
```

Comment:
All scripts should be in the `scripts` folder using ESModule syntax. Don't forget to add it to the list of entrypoints in the rollup config